src/content/general/management-clusters/index.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: Management clusters
+title: Management clusters
+description: The role Management Clusters perform in the Giant Swarm architecture and how they are updated.
+weight: 50
+menu:
+  main:
+    identifier: management-clusters
+    parent: general
+last_review_date: 2021-02-17
+user_questions:
+  - How do Giant Swarm use Management Clusters?
+  - How are Management Clusters kept up to date?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-biscuit
+---
+
+## Overview
+
+As we are fully convinced of Kubernetes as a platform for building platforms, we build all our management clusters based on Kubernetes. Giant Swarm leverages the concept of [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to control all resources that clusters need as [“Custom Resources”](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+As well as operators, our management clusters run supporting components such as our monitoring stack and Happa, our web UI. All components are deployed using the Giant Swarm [App Platform]({{< relref "/app-platform/overview" >}}).
```

Comment:
```suggestion
As well as operators, our management clusters run supporting components such as our monitoring stack and our web UI. All components are deployed using the Giant Swarm [App Platform]({{< relref "/app-platform/overview" >}}).
```

Reply:
Towards getting rid of Happa from the documentation.

Reply:
Bonus points for linking `web UI` to `/ui-api/web/` :)